### PR TITLE
feat: add convenient `run:fast` task and update readme

### PR DESCRIPTION
**Pull request by sigmaSd**

### CODE REVIEW

**Comment by sigmaSd** [comment]
Maybe in the future run task could detect if there was changes to webview library automatically instead of adding another task for this.

File: deno.json [hunk 1]
```diff
@@ -5,7 +5,8 @@
     "fmt:check": "deno fmt --unstable --check",
     "lint": "deno lint --unstable",
     "build": "deno run -A --unstable script/build.ts",
-    "run": "deno task build && export PLUGIN_URL=\"./build/\" && deno run -A --unstable"
+    "run": "deno task build && export PLUGIN_URL=\"./build/\" && deno run -A --unstable",
+    "fast-run": "export PLUGIN_URL=\"./build/\" && deno run -A --unstable"
```
Comment (eliassjogreen) [inline deno.json:9]: To match the rest of the tasks:

```suggestion
    "run:fast": "export PLUGIN_URL=\"./build/\" && deno run -A --unstable"
```

File: README.md [hunk 2]
```diff
@@ -97,6 +102,13 @@ or
 $ deno task run examples/local.ts
 ```
 
+or if you have the webview library already built and didn't make any changes to
+it, you can skip the building step with
+
+```bash
+$ deno task fast-run examples/local.ts
```
Comment (eliassjogreen) [inline README.md:109]:
```suggestion
$ deno task run:fast examples/local.ts
```

**Review by eliassjogreen — CHANGES_REQUESTED** [review]
LGTM apart from one nitpick

**Comment by sigmaSd** [comment]
wait I messed up git

**Pull request merged**